internal/inc/file_internal.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef FILE_INTERNAL_H

```

Comment:
>FILE_INTERNAL_H [](start = 8, length = 15)

Why do we need this internal? Sorry if I missed explanations as we went through the review.

Reply:
These are some structs that are used for implementation. Since they are common between linux and windows, we put them in an internal folder.

---
In reply to: [451982465](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451982465) [](ancestors = 451982465)